src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,120 @@
+# Tuning Component Settings
+
+Helm Charts are a set of Kubernetes manifests that reflect best practices for deploying an
+application or service. Helm is heavily influenced by [Homebrew](http://brew.sh/), including the
+[formula model](https://github.com/Homebrew/homebrew-core). A Helm Chart is to Helm as a Formula
+is to Homebrew.
+
+After you run `helmc fetch deis/workflow-rc2`, you can customize the chart using
+`helmc edit workflow-rc2` before using `helmc generate workflow-rc2` and
+`helmc install workflow-rc2` to complete the installation. To customize the respective component,
+edit `manifests/deis-<component>-rc.yaml` and modify the `env` section of the component to tune
+these settings.
+
+For example, to allow only administrators to register new accounts in the controller,
+edit `manifests/deis-controller-rc.yaml` and add the following under the `env` section:
+
+```
+env:
+  - name: REGISTRATION_MODE
+    value: "admin_only"
+```
+
+## Customizing the Builder
+
+The following environment variables are tunable for the [Builder][] component:
+
+Setting | Description
+------- | ---------------------------------
+DEBUG   | Enable debug log output (default: false)
+
+## Customizing the Controller
+
+The following environment variables are tunable for the [Controller][] component:
+
+Setting                                         | Description
+----------------------------------------------- | ---------------------------------
+REGISTRATION_MODE                               | set registration to "enabled", "disabled", or "admin_only" (default: "enabled")
+GUNICORN_WORKERS                                | number of [gunicorn][] workers spawned to process requests (default: num_cpus * 2 + 1)
```

Comment:
This got changed in deis/controller#833 (sorry!), so let's use the same text as in #342:
`CPU cores * 4 + 1`